Makefile

```diff
@@ -0,0 +1,47 @@
+GO111MODULE=on
+
+GOARCH?=$(shell go env GOARCH)
+GOOS?=$(shell go env GOOS)
+GOLANGCI_VERSION?=v1.24.0
+VERSION?=$(shell cat VERSION)
+DOCKER_REPO?=quay.io/dgrisonnet/kube-events-exporter
+
+.PHONY: all
+all: lint build test
+
+.PHONY: lint
+lint: check-license
+	@docker run --rm -v $(shell pwd):/app:ro \
+					 -w /app \
+					 golangci/golangci-lint:$(GOLANGCI_VERSION) \
+					 golangci-lint run -v
+
+.PHONY: check-license
+check-license:
+	./scripts/check_license.sh
+
+.PHONY: build
+build: kube-events-exporter
+
+.PHONY: kube-events-exporter
+kube-events-exporter:
+	CGO_ENABLED=0 go build -ldflags "-s -w"
```

Comment:
Could we have this be GOOS specific, so we can easily override it when building for other distros. Thanks! 
https://github.com/kubernetes/kube-state-metrics/blob/master/Makefile#L63

Reply:
Good catch, I forgot to add them here